README.md

```diff
@@ -1 +1,10 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+> This is a project to test the basics of creating a new repository.
+
+## Built WIth
+- Html
+- Css
+
+## Authors:
+:bust_in_silhouette:Gabriel Fonseca
+- Github: @Gopxfs
```

Comment:
- [ ] Please make your readme file like the Microverse README template to make it more descriptive.
- Add a screenshot of your webpage.
- Getting Start section like if someone cloned your repo how should he run it.
- In the author section add a link to your Github profile and a link to your Linkedin profile.
- Show your support section.
- Acknowledgments section.

Reply:
Hey @ShahierNashaat ! Did everything you told! Thanks.